packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -266,6 +300,7 @@ export class LocalIntervalCollection<TInterval extends ISerializableInterval> {
     private readonly endIntervalTree: MergeTree.RedBlackTree<TInterval, TInterval>;
     private conflictResolver: MergeTree.IntervalConflictResolver<TInterval>;
     private endConflictResolver: MergeTree.ConflictAction<TInterval, TInterval>;
+    private nextIntervalID: number = 0;
```

Comment:
an integer like this can't be easily made eventually consistent across clients. it's probably easiest to go with a string and use uuid (lots of examples in the codebase) to generate a globally unique id.

Reply:
Makes sense. Thanks.